snippets/base/static/js/admin-publish.js

```diff
@@ -0,0 +1,11 @@
+django.jQuery(document).ready(function() {
+    django.jQuery('[data-tool-name="publish_object"]').on("click", function(event) {
+        function confirmation() {
```

Comment:
For my own curiosity: why did you need to put this in a nested function and then immediately call it?

Reply:
because i suck at javascript? :D